packages/relay/tests/lib/sdkClient.spec.ts

```diff
@@ -2232,7 +2235,11 @@ describe('SdkClient', async function () {
         .resolves(getTransactionResponse('FileDeleteTransaction'));
 
       hbarLimitMock.expects('addExpense').withArgs(fileDeleteFee).once();
-      hbarLimitMock.expects('shouldLimit').never();
+      hbarLimitMock
+        .expects('shouldLimit')
+        .withArgs(sinon.match.any, SDKClient.transactionMode, callerName)
+        .once()
+        .returns(false);
```

Comment:
We don't want to do any rate limiting on `FileDeleteTransaction`, we always want to clean up files so we don't want the rate limiter to block us in this case. 

It was discussed [here](https://github.com/hashgraph/hedera-json-rpc-relay/pull/2756#discussion_r1691468790).

Reply:
oh yeah nice catch! Nana and I also had a conversation about this I must've forgotten about this since the shouldLimit check is in executeTransaction() and deleteFile() now uses executeTransaction(). Will push in a solution soon!